.env-template

```diff
@@ -100,3 +97,19 @@ NODE_NAME=__YOUR_NODE_NAME_GOES_HERE__
 
 # Yagna service REST API application key token.
 YAGNA_APPKEY=__GENERATED_APP_KEY__
+
+# Choose NET type to use. Central NET is used by default, but will be removed
+# in future version to use more decentralized solutions.
+#YA_NET_TYPE=hybrid
```

Comment:
Why are the env variable changes also mixed into the p2p direct messaging changes?

This makes the PR a lot less clean.

Did you also update GOTH to be compatible with the env change?

Reply:
There are no changes to ENV themselves. I added new variables, but didn't change any. Here I added the documentation only.

There is a change in resolving relay server address and it was necessary for yagna to work.
Goth passes as you probably saw ;)

Reply:
ahh, i was a bit too quick on the comment there, sorry.

Yes, goth seems to work so its good :)